lib/modules/commentSortBy.js

```diff
@@ -0,0 +1,52 @@
+/* @flow */
+
+import _ from 'lodash';
+import { $ } from '../vendor';
+import { Module } from '../core/module';
+import { rawLocale } from '../utils/localization';
+import {
+	string,
+	watchForRedditEvents,
+} from '../utils';
+
+export const module: Module<*> = new Module('commentSortBy');
+
+module.beforeLoad = () => {
+	watchForRedditEvents('postModTools', () => {
```

Comment:
Could you pass along the `postModTools` JSAPI container to `applyToPost`?

Reply:
🏁